### PR TITLE
[Backport][ipa-4-9] Add support subordinate user/group ids as separate entries

### PR DESCRIPTION
**Pull request by fcami**
This is a MANUAL BACKPORT of https://github.com/freeipa/freeipa/pull/5713

### CODE REVIEW

**Comment by flo-renaud** [comment]
Hi @fcami 
azure detected lint issues:
```
************* Module ipaserver.install.ldapupdate
ipaserver/install/ldapupdate.py:90: [E0602(undefined-variable), get_sub_dict] Undefined variable 'replication')
ipaserver/install/ldapupdate.py:92: [E0602(undefined-variable), get_sub_dict] Undefined variable 'replication')
ipaserver/install/ldapupdate.py:96: [E0602(undefined-variable), get_sub_dict] Undefined variable 'replication')

```

**Comment by fcami** [comment]
Yes, in short it needs part of, or all of https://github.com/freeipa/freeipa/commit/16e1cbdc5e2192e47f11df4148069a8b7e2220f7 backported too

File: ipatests/prci_definitions/gating.yaml [hunk 1]
```diff
@@ -298,3 +298,15 @@ jobs:
         template: *ci-ipa-4-9-latest
         timeout: 3600
         topology: *master_1repl
+
+  fedora-latest/test_subids:
```
Comment (flo-renaud) [inline ipatests/prci_definitions/gating.yaml:302]: To follow the implicit naming convention, the task should be fedora-latest-ipa-4-9, and it requires [fedora-latest-ipa-4-9/build], with build_url '{fedora-latest-ipa-4-9/build_url}' + template *ci-ipa-4-9-latest

Comment (fcami) [reply ipatests/prci_definitions/gating.yaml]: Fixed, thanks Flo!

**Comment by fcami** [comment]
Azure CI error:
"ipatests.pytest_ipa.integration.host.Host.client1.cmd98:transport.py:557 Error: Failed to download metadata for repo 'fedora': Cannot prepare internal mirrorlist: Curl error (6): Couldn't resolve host name for https://mirrors.fedoraproject.org/metalink?repo=fedora-34&arch=x86_64 [Could not resolve host: mirrors.fedoraproject.org]"

**Comment by rcritten** [comment]
LGTM. The differences between this and master are between test names and the lack of autobind in ipa-4-9.

**Comment by rcritten** [comment]
ipa-4-9:

* 3540986a11d4f3401ba4918f25229a79283d9dbd Add basic support for subordinate user/group ids
* 5d4fe06663c3e66b1da73c01ce022790634a3e3b Redesign subid feature
* ef115b04182d572bf61e32e2405bbb68ff65e928 Use 389-DS' dnaInterval setting to assign intervals
* e6e3fb606d08b0dc57bfa360a0f0082052441db6 Fix ipa-server-upgrade
* 44ccc0f64bac9fc2e7e3264984af26635bb34742 Fix oid of ipaUserDefaultSubordinateId
* 9f4b8982cd06011df8daac480a726637fc52649e WebUI: Improve subordinate ids user workflow
* b53a52a1fafa94e0129e6e3e55fddd59909f0f0a Test DNA plugin configuration

**Pull request closed without merging**